csharp/private/rules/nunit_test.bzl

```diff
@@ -39,6 +39,10 @@ def _nunit_test_impl(ctx):
     result = providers.values()
     result.append(DefaultInfo(
         executable = result[0].out,
+        runfiles = ctx.runfiles(
+            files = [result[0].out, result[0].pdb],
+            transitive_files = result[0].transitive_runfiles,
+        ),
```

Comment:
There is a decent amount of duplication in the rule impls. I've been delaying refactoring that before it's obvious to me what the shared API should look like. Might be time to do it 🤷‍♂